python/openvino_tokenizers/tokenizer_pipeline.py

```diff
@@ -892,10 +892,30 @@ def from_hf_json_template_postprocessor(
         cls, post_processor_dict: Dict[str, Any], number_of_inputs: int = 1, add_special_tokens: bool = True
     ) -> "CombineSegmentsStep":
         inputs: List[TokenWithTypeId] = []
-        if number_of_inputs == 1:
-            post_processor = post_processor_dict["single"]
-        else:
-            post_processor = post_processor_dict["pair"]
+        
+        post_processor = post_processor_dict["single"]
+        pair_post_processor = post_processor_dict["pair"]
+        
+        single_num_inputs = len(post_processor)
+        pair_num_inputs = len(pair_post_processor)
+        num_additional = pair_num_inputs - single_num_inputs
+        start_from_idx = single_num_inputs - num_additional
+        
+        # We support only adding one additional pair for the second input
+        assert num_additional in [2] and start_from_idx >= 0
```

Comment:
```suggestion
        assert num_additional == 2 and start_from_idx >= 0
```

Reply:
Better to avoid `assert` and raise a human-readable exception.

Reply:
done